src/Security/Authentication/Negotiate/src/PACKAGE.md

```diff
@@ -0,0 +1,49 @@
+## About
+
+`Microsoft.AspNetCore.Authentication.Negotiate` contains middleware to authenticate requests using Windows Authentication (also known as Negotiate, Kerberos, or NTLM).
+
+## How to Use
+
+To use `Microsoft.AspNetCore.Authentication.Negotiate`, follow these steps:
+
+### Installation
+
+```shell
+dotnet add package Microsoft.AspNetCore.Authentication.Negotiate
+```
+
+### Configuration
+
+To use the middleware, configure it in your ASP.NET Core app's `Program.cs`:
+
+```csharp
+builder.Services.AddAuthentication(NegotiateDefaults.AuthenticationScheme)
+   .AddNegotiate();
+
+builder.Services.AddAuthorization(options =>
+{
+    options.FallbackPolicy = options.DefaultPolicy;
+});
+
+var app = builder.Build();
+
+app.UseAuthentication();
+app.UseAuthorization();
+```
```

Comment:
```suggestion
```csharp
builder.Services.AddAuthentication(NegotiateDefaults.AuthenticationScheme)
   .AddNegotiate();

builder.Services.AddAuthorization();

var app = builder.Build();
```
```

Unless we're planning on showing `options.FallbackPolicy = options.DefaultPolicy;` in every authentication package README, I wouldn't do it here. `UseAuthentication` and `UseAuthorization` will also get added for you manually these days if you call `AddAuthorization()`.